### ISSUE TITLE
Can't update list of records

### ISSUE THREAD

**Issue by pushred** [prisma/prisma-admin-feedback#158]
![prisma-admin-list](https://user-images.githubusercontent.com/431435/64432016-a216a200-d089-11e9-839a-a0c0d0ec2e76.gif)

```TypeError: Cannot read property 'split' of undefined
    at Object.t.parsePathId (app.js:1)
    at a (app.js:1)
    at app.js:1
    at app.js:1
    at t.a (app.js:1)
    at app.js:1
    at Object.t.a (app.js:1)
    at Object.t.updateMutations (app.js:1)
    at Object.t.getMutations (app.js:1)
    at app.js:1
    at app.js:1
    at Object.next (app.js:1)```

**Comment by itsbentsai** [comment]
Have the exact same issue. Is there a solution to this?

**Comment by Sandburg** [comment]
Same here. Would be very grateful for a solution on this issue.

**Comment by kanblack** [comment]
Same here

**Comment by mikeThurmond** [comment]
Has anyone on here made any progress? Does writing the mutation work?

**Comment by stefanfuchs** [comment]
I have tested different versions of the prisma docker image, and here is what I found out:

- 1.34.0 ok
- 1.34.5 ok
- 1.34.6 ok
- 1.34.7 not ok (token error)
- 1.34.8 not ok
- 1.34.9 not ok
- 1.34.10 not ok

**Comment by swdove** [comment]
I'm also encountering this issue in 1.34.10 with two relational tables:

type User {
  id: ID! @id
  contactId: String! @unique
  email: String! @unique
  firstName: String!
  lastName: String!
  registrationToken: String @unique
  registrationTokenExpires: DateTime
  passwordResetToken: String @unique
  passwordResetTokenExpires: DateTime
  webhookClients: [WebhookClient] @relation(name: "WebhookClientsUsers")
  createdAt: DateTime! @createdAt
  updatedAt: DateTime! @updatedAt
}

type WebhookClient {
  id: ID! @id
  tagId: Int!
  deliveryUrl: String!
  isActive: Boolean!
  clientUsers: [User] @relation(name: "WebhookClientsUsers")
  createdAt: DateTime! @createdAt
  updatedAt: DateTime! @updatedAt
}

The Prisma UI will allow me to add webhookClients to a User or clientUsers to a Webhook but won't allow me to save those changes, displaying the "Cannot read property 'split' of undefined" javascript error.

**Comment by crucialfelix** [comment]
I've tried going from 1.34.10 back to 1.34.6, but now the admin hangs forever at `loading...`. 

Maybe something internal isn't backwards compatible.

Going back to 1.34.10 I can load the admin again.

**Comment by stefanfuchs** [comment]
Have you tried clearing the browser's local storage and doing a shift+f5
(Ctrl+f5 on Firefox)?

**Comment by mghabdo** [comment]
Stil the problem not resolved ?!

**Comment by wilomgfx** [comment]
same here :(

**Comment by musbenlahrech** [comment]
I have similar issue even with 1.34.6 version 
I am just frustrated with it any fix ?

**Comment by stefanfuchs** [comment]
@musbenlahrech if you are using v1.34.6 try clearing the browser's local storage and doing a shift+f5; Your browser might be using old cached prisma admin files and/or data.

**Comment by stefanfuchs** [comment]
But still, I think this project should be open-sourced. I have tried finding the prisma admin source code, but seems to not be available anywhere.
The developers don't seem to be paying enough attention to solve small issues like this. We could solve a bug like this easily if we had access to the code.

**Comment by musbenlahrech** [comment]
> @musbenlahrech if you are using v1.34.6 try clearing the browser's local storage and doing a shift+f5; Your browser might be using old cached prisma admin files and/or data.

I could load it but sometimes create doesn't work with nested data

**Comment by musbenlahrech** [comment]
> But still, I think this project should be open-sourced. I have tried finding the prisma admin source code, but seems to not be available anywhere.
> The developers don't seem to be paying enough attention to solve small issues like this. We could solve a bug like this easily if we had access to the code.

Yes definitely, it's is a really important part of the stack since it saves a looot of time and it's why we picked prisma from the beginning.

**Comment by stefanfuchs** [comment]
> > @musbenlahrech if you are using v1.34.6 try clearing the browser's local storage and doing a shift+f5; Your browser might be using old cached prisma admin files and/or data.
> 
> I could load it but sometimes create doesn't work with nested data

Creating nested data does not work for me either. I need to create each part individually otherwise an error occurs.
